packages/foundation-legacy/etc/foundation-legacy.api.md

```diff
@@ -9,13 +9,13 @@ import { IStyle } from '@fluentui/style-utilities';
 import { IStyleSet } from '@fluentui/style-utilities';
 import { ITheme } from '@fluentui/style-utilities';
 import { styled as legacyStyled } from '@fluentui/utilities';
-import * as React from 'react';
+import * as React_2 from 'react';
```

Comment:
API Extractor made this change automatically, and AFAIK we haven't been able to determine why (or whether it points to a real issue). My best guess is that *something somewhere* in the compilation includes an additional reference to `React`, possibly a different version of `@types/react`, which is hidden because we do `skipLibCheck` everywhere.